addons/position_manager/position_manager.lua

```diff
@@ -0,0 +1,83 @@
+--Copyright (c) 2020, Lili
+--All rights reserved.
+
+--Redistribution and use in source and binary forms, with or without
+--modification, are permitted provided that the following conditions are met:
+
+--    * Redistributions of source code must retain the above copyright
+--      notice, this list of conditions and the following disclaimer.
+--    * Redistributions in binary form must reproduce the above copyright
+--      notice, this list of conditions and the following disclaimer in the
+--      documentation and/or other materials provided with the distribution.
+--    * Neither the name of position_manager nor the
+--      names of its contributors may be used to endorse or promote products
+--      derived from this software without specific prior written permission.
+
+--THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+--ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+--WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+--DISCLAIMED. IN NO EVENT SHALL Lili BE LIABLE FOR ANY
+--DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+--(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+--LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+--ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+--(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+--SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+_addon.name = 'position_manager'
+_addon.author = 'Lili'
+_addon.version = '1.0.0'
+_addon.command = 'pm'
+
+if not windower.file_exists(windower.windower_path .. '\\plugins\\WinControl.dll') then
+    print('position_manager: error - Please install the WinControl plugin in the launcher.')
+    windower.send_command('lua u position_manager')
+    return
+end
+
+config = require 'config'
+
+default = {
+    x = 0,
+    y = 0,
+}
+
+settings = config.load(default)
+
+function move(settings)
+    windower.send_command('load wincontrol')
+    coroutine.sleep(0.5)
+    windower.send_command('wincontrol move %s %s':format(settings.x,settings.y))
+    coroutine.sleep(0.5)
+    windower.send_command('unload wincontrol')
+end
+
+function handle_commands(cmd,name,pos_x,pos_y)
+    local cmd = cmd or 'help'
```

Comment:
Can actually omit the `local` here, it will still be local since it only adjusts itself. And while not necessary, it is quite customary to be case-insensitive for v4 addons, this is a common pattern:
```lua
cmd = cmd and cmd:lower() or 'help'
```

Reply:
Fixed.